src/middlewares/Validations/passwordValidator.js

```diff
@@ -0,0 +1,49 @@
+import errorResponse from '../../utils/index';
+import { decodeToken } from '../../helpers/jwtHelper';
+
+/**
+ *
+ * @param {Object} req
+ * @param {Object} res
+ * @param {function} next
+ * @returns {(function|Object)} function next() or an error response object
+ * @description Handles validation for the password reset endpoint
+ */
+const checkForValidPassword = (req, res, next) => {
+  const { query } = req.query;
+  const { password, confirmPassword } = req.body;
+
```

Comment:
This password validator, don't you think you should also add it to the sign up route. If you all really want to validate password in your app

Reply:
@girlking is using a validation package for the signup route, But we can find a way to work with it.

Reply:
Yes she is but she is not validating all this for the user to sign up,  We can look into this later. Well done